### ISSUE TITLE
how can we get lat and lng from marker?

### ISSUE THREAD

**Issue by LittleFiveSmile** [OpenCageData/leaflet-opencage-geocoding#14]
Hi
I try to store the lat and lng after searching by leaflet-opencage-search. It just show a marker on the map, but how can we get lat and lng of that marker?

Thanks

**Comment by mtmail** [comment]
We released version 1.3.0 now and updated the README file.

In the `options` you can set `onResultClick`. For example to
```javascript
function(result){ console.log(result.center.lng, result.center.lat, result.name); }
```

**Comment by LittleFiveSmile** [comment]
> We released version 1.3.0 now and updated the README file.
> 
> In the `options` you can set `onResultClick`. For example to
> 
> ```js
> function(result){ console.log(result.center.lng, result.center.lat, result.name); }
> ```

Thanks a lot. :)
